apps/api/src/db/index.ts

```diff
@@ -18,17 +18,20 @@
  */
 
 import { drizzle } from 'drizzle-orm/node-postgres';
+import { setStatus, Status } from '../app-health.js';
 
 export type PostgresDb = ReturnType<typeof drizzle>;
 
 export const connectToDb = (connectionString: string): PostgresDb => {
 	try {
 		const db = drizzle(connectionString);
 
+		setStatus('db', { status: Status.OK });
 		return db;
 	} catch (err) {
 		console.log('Error on Database startup');
 		console.log(err);
+		setStatus('db', { status: Status.ERROR, info: { err } });
 		throw err;
```

Comment:
OK so on review I would recommend updating the catch statement to the following:

```
		if (err instanceof Error) {
			setStatus('db', { status: Status.ERROR, info: { message: err.message } });
		} else {
			const error = String(err);
			setStatus('db', { status: Status.ERROR, info: { message: error } });
		}
```

Reference: https://kentcdodds.com/blog/get-a-catch-block-error-message-with-typescript
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch#catch_binding

This way we can insure error message is displayed in Swagger / Health check, and we can update the `info` type accordingly